sqlglot/expressions.py

```diff
@@ -1190,6 +1190,10 @@ def ctas(self, table, properties=None, dialect=None, parser_opts=None, copy=True
     def named_selects(self):
         return [e.alias_or_name for e in self.args["expressions"] if e.alias_or_name]
 
+    @property
+    def selections(self):
```

Comment:
why not selects?

Reply:
Ah sure

Reply:
FWIW: All these properties have gotten a little confusing... some return strings, some return Expressions.